scripts/ci/create-plunker-bundle.js

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env node
+
+'use strict';
+
+/*
+ * This script creates a bundle of all components and publishes it to Firebase.
+ * The bundle will be used to load a Plunker Demo of Angular Material 2.
+ */
+
+const spawn = require('child_process').spawnSync;

```

Comment:
Ah gotcha. I thought that Jeremy meant, that it shouldn't run in other branches. Will add a check for the environment variable.